sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/AlreadyExistException.scala

```diff
@@ -56,7 +56,10 @@ class NamespaceAlreadyExistsException private(
   }
 
   def this(message: String) = {
-    this(message, errorClass = None, messageParameters = Map.empty[String, String])
+    this(
+      message,
+      errorClass = Some("SCHEMA_ALREADY_EXISTS"),
```

Comment:
It is a bit unfortunate we have to use the same string literal in multiple places. Adding a constant would require a companion object and an import as these values are accessed in constructors. I was not sure it would be worth it so kept as is.